### ISSUE TITLE
bleeding call forward error

### ISSUE THREAD

**Issue by manfer** [irontec/ivozprovider#503]
I'm testing bleeding standalone installation.

When I configure a call forward inconditional on a user it works fine when he receives a call from another user. But if the user is the target of a DDI and I make an incomming call to that DDI then the call fails.

The only debug message I have been able to gather is from asterisk in verbose mode:

```
ERROR[1688][C-0000000e]: utils.c:1441 ast_carefulwrite: write() returned error: Connection reset by peer
```

This comes just after agi funcions in CallForwardAction had set some values:

```
[b2a69ae2] [CallForwardAction.php:145] AGI SetVar("PJSIP/proxytrunks-00000015", "REDIRECTING(from-num,i) = +34XXXXXXXXX")
[b2a69ae2] [CallForwardAction.php:146] AGI SetVar("PJSIP/proxytrunks-00000015", "REDIRECTING(from-tag,i) = 13")
[b2a69ae2] [CallForwardAction.php:147] AGI SetVar("PJSIP/proxytrunks-00000015", "REDIRECTING(from-name) = John Doe")
```

**Comment by manfer** [comment]
I have enabled agi debug

```
<PJSIP/proxytrunks-0000001b>AGI Rx << SET VARIABLE REDIRECTING(from-num,i) "+34XXXXXXXXX" CallForwardAction.php:145
       > [7974fe33] [CallForwardAction.php:145] AGI SetVar("PJSIP/proxytrunks-0000001b", "REDIRECTING(from-num,i) = +34XXXXXXXXX")
<PJSIP/proxytrunks-0000001b>AGI Tx >> 200 result=1
<PJSIP/proxytrunks-0000001b>AGI Rx << SET VARIABLE REDIRECTING(from-tag,i) "13" CallForwardAction.php:146
       > [7974fe33] [CallForwardAction.php:146] AGI SetVar("PJSIP/proxytrunks-0000001b", "REDIRECTING(from-tag,i) = 13")
<PJSIP/proxytrunks-0000001b>AGI Tx >> 200 result=1
<PJSIP/proxytrunks-0000001b>AGI Rx << SET VARIABLE REDIRECTING(from-name) "John Doe" CallForwardAction.php:147
       > [7974fe33] [CallForwardAction.php:147] AGI SetVar("PJSIP/proxytrunks-0000001b", "REDIRECTING(from-name) = John Doe")
<PJSIP/proxytrunks-0000001b>AGI Tx >> 200 result=1
<PJSIP/proxytrunks-0000001b>AGI Rx << GET VARIABLE CALLER
<PJSIP/proxytrunks-0000001b>AGI Tx >> 200 result=1 (User#3)
<PJSIP/proxytrunks-0000001b>AGI Rx << GET VARIABLE ORIGIN
<PJSIP/proxytrunks-0000001b>AGI Tx >> 200 result=0
<PJSIP/proxytrunks-0000001b>AGI Rx << PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Notice: Undefined offset: 1 in /opt/irontec/ivozprovider/asterisk/agi/src/Agi/ChannelInfo.php:139
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << Stack trace:
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
[May 27 23:40:25] ERROR[3194][C-00000012]: utils.c:1441 ast_carefulwrite: write() returned error: Connection reset by peer
<PJSIP/proxytrunks-0000001b>AGI Rx << #0 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/ChannelInfo.php(126): Agi\ChannelInfo->getChannelData('ORIGIN')
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #1 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/ExternalUserCallAction.php(72): Agi\ChannelInfo->getChannelOrigin()
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #2 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/RouterAction.php(398): Agi\Action\ExternalUserCallAction->process()
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #3 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/RouterAction.php(339): Agi\Action\RouterAction->routeToExternal()
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #4 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/CallForwardAction.php(155): Agi\Action\RouterAction->route()
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #5 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/UserStatusAction.php(137): Agi\Action\CallForwardAction->process()
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
<PJSIP/proxytrunks-0000001b>AGI Rx << #6 /opt/irontec/ivozprovider/asterisk/agi/src/Agi/Action/UserStatusAction.php(102): Agi\Action\UserStatusAction->proce in /opt/irontec/ivozprovider/asterisk/agi/src/Agi/ChannelInfo.php on line 139
<PJSIP/proxytrunks-0000001b>AGI Tx >> 510 Invalid or unknown command
    -- <PJSIP/proxytrunks-0000001b>AGI Script agi://127.0.0.1:4573/fastagi-runner.php?command=dialplan/trunks completed, returning 0
<PJSIP/proxytrunks-0000001b>AGI Tx >> HANGUP
    -- Auto fallthrough, channel 'PJSIP/proxytrunks-0000001b' status is 'UNKNOWN'
```

**Comment by Kaian** [comment]
Fixed in #509 
Thanks for reporting!
